frontend/cypress/e2e/mobileviewport.spec.js

```diff
@@ -0,0 +1,18 @@
+describe('Tabela de Clientes - Exibição de colunas em viewport móvel', () => {
+    beforeEach(() => {
+      cy.visit('http://localhost:3000');
+  
+      // Simula viewport do Galaxy S20 Ultra
+      cy.viewport(412, 915);
+    });
+  
+    it('Mostra as colunas "Nome da Empresa" e "Ação" e oculta as outras colunas', () => {
+      cy.get('th').contains('Company name').should('be.visible');
```

Comment:
Uma solução mais simples seria:

```suggestion
      cy.contains('th', 'Company name').should('be.visible');
```

O mesmo vale para as próximas linhas.